src/server/rest/v1/service/TenantService.ts

```diff
@@ -381,6 +381,46 @@ export default class TenantService {
     next();
   }
 
+  public static async handleUpdateTenantCoordinations(action: ServerAction, req: Request, res: Response, next: NextFunction): Promise<void> {
+    const filteredRequest = TenantValidatorRest.getInstance().validateTenantUpdateCoordinationsReq(req.body);
+    // Check auth
+    if (!await Authorizations.canUpdateTenant(req.user)) {
+      throw new AppAuthError({
+        errorCode: HTTPAuthError.FORBIDDEN,
+        user: req.user,
+        action: Action.UPDATE, entity: Entity.TENANT,
+        module: MODULE_NAME, method: 'handleUpdateTenantCoordinations',
+        value: filteredRequest.id
+      });
+    }
+    // Get
+    const tenant = await TenantStorage.getTenant(filteredRequest.id);
```

Comment:
Apply projected field you are returning the full object

Reply:
As discussed, this will be available after the tenants auth migration.